_posts/2023-08-20-my-lfx-mentorship-experience-with-the-cncf-strimzi-community.md

```diff
@@ -0,0 +1,170 @@
+With my interest piqued by this innovative endeavor, I decided to join the Strimzi community through the LFX Mentorship program.
+
+The LFX mentorship program bridges budding developers and experienced mentors, providing an avenue to contribute meaningfully to open-source projects.
+
+# From where everything started
+
+Everything started in 2022, I was looking for things I could do during my summer holidays.
+I investigated some mentorship programs such as Google Summer of Code, MLH Fellowship, Outreachy, and LFX Mentorship.
+
+## The Application process
+
+I have selected two projects for which my skill set fits the recommended skills.
+Among them, the Strimzi project was there waiting for me. 
+
+`“CNCF - Strimzi: Proof of Concept of an MQTT to Apache Kafka bridge for producing messages.”`
+
+This project got my attention among all the projects because I had practical experience with MQTT from my undergraduate program.
+I learned the basic concepts of Apache Kafka from the Cloud Computing and Distributed System course at IIIT Delhi.
+
+### The cover letter and the résumé
+
+After understanding what I had to do and what the project was about, I had to prepare my cover letter and resume.
+You need to submit these essential documents while applying to the LFX mentorship.
+I ensured my cover letter and resume were well-aligned with the project by providing details about my skills and experience with the recommended skills.
+
+But just after submitting my application, I noticed that one of the mentors for my project commented on the weak selection process.
+He was asking if they could ask for a proposal just like in Google Summer of Code.
+
+<!--Image here -->
+
+This scared me :) But I was confident I could provide a good proposal because of my previous experience with the underlying technologies.
+
+On the same day, I got my first interaction with my mentors, I would never think it wouldn’t be the last one.
+
+<!--Image here -->
+
+They gave me two days to elaborate on the proposal, and to be honest, it was the first time I saw myself as a real engineer.
+
+The first task was understanding the real problem and why we would need to build such a bridge, so I researched that.
+To design the solution, my first intuition was to relate my college project to the project.
+Just to get a rough idea of how it is very similar to what I was building in college:
+
+<!--Image here -->
+
+After that, I started to think about the architecture of the solution, and I came up with this:
+
+<!--Image here -->
+
+To be even closer to a logical solution, the win-win was to study and understand how Strimzi HTTP-Kafka Bridge works and how it is deployed on Kubernetes. 
+After reading the Strimzi HTTP-Kafka Bridge documentation, I was able to design a Bridge tackling all the issues. 
+Of course, it was not perfect, and this is where discussions and critical thinking helped me.
+
```

Comment:
I would merge "From where everything started", "The Application process" and "The cover letter and the résumé" sections in just one. The first two are really small to deserve a title by themselves imho.
I would have just one maybe titled "The application process".